.gitlab-ci.yml

```diff
@@ -893,7 +898,15 @@ test_backend_integration:
     reports:
       junit: results_backend_integration_*.xml
 
-test_full_integration:
+test_backend_integration_open:
+  extends: test_backend_integration_enterprise
+  only:
+    variables:
+      - $RUN_INTEGRATION_TESTS == "true"
```

Comment:
I'm not really sure if this is inherited from parent job or not. And also thought that maybe we need to have it for each job. Please advice maybe this `only` section can be completely removed.

Reply:
I'm pretty sure it is inherited.

Reply:
Yes, we don't need this.